tests/components/test_grid_component.py

```diff
@@ -98,9 +98,6 @@ def test_create_grid_from_bbox_rotated(mock_model):
     assert "xc" in grid_component.data.coords
     assert grid_component.data.raster.y_dim == "y"
     assert np.isclose(grid_component.data.raster.res[0], 0.05)
-    # Check set_geoms call
-    geom = grid_component._model.method_calls[0][1][0]
-    assert isinstance(geom, gpd.GeoDataFrame)
```

Comment:
You did update that region would be added to geoms though no? Can you check that here?

Reply:
I have lost track of the discussion around the region somewhat but my understanding was that the region(s) wouldn't be part of the geoms component anymore since they have their own specialised behaviour

Reply:
I'll be merging this one now, but if you think this needs changing we can pick that up in the region PR from Marnix as we do the rest of the things you mentioned.